testing/integration-tests/src/lib.rs

```diff
@@ -5,27 +5,32 @@
 #![deny(unused_crate_dependencies)]
 
 #[cfg(test)]
-mod codegen;
-#[cfg(test)]
-mod utils;
+pub mod utils;
 
-#[cfg(test)]
-mod blocks;
-#[cfg(test)]
-mod client;
-#[cfg(test)]
-mod frame;
-#[cfg(test)]
-mod metadata;
-#[cfg(test)]
-mod runtime_api;
-#[cfg(test)]
-mod storage;
+// We expose the utils crate's contents as public to avoid
+// a few unused warnings when running the tests under the
+// "unstable-light-client" flag.
+#[cfg(all(test, feature = "unstable-light-client"))]
+pub use utils::*;
```

Comment:
I'd probably either:

1. Just always expose as pub; the less switching based on feature flags the better IMO (every feature flag usage makes our lives a little more complicated), or
2. Don't change the visibility (it's a bit of a hack) and just add the relevant `#[cfg(test, allow(unused)]` type thing (whatever it is) if that works; it more directly addresses the actual issue so feels clearer to me :)